resources/js/components/modules/templates/videoTemplate.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div class="video-template row">
```

Comment:
Could you add a class of `mb-5` to be more aesthetically pleasing with the space between the footer and the component on desktop